OsmAnd/src/net/osmand/plus/views/MapQuickActionLayer.java

```diff
@@ -73,6 +74,7 @@
     private boolean isLayerOn;
 
     private boolean nightMode;
+    private boolean prevWidgetState;
```

Comment:
Meaning is not clear, probably should be removed

Reply:
This should stay but properly renamed, actually previous naming "showWidget" is also not clear and better naming should be used. It is not  "previous", I guess it is "current" state

Reply:
renamed it to currentWidgetState